Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM debian:jessie
-RUN apt-get update && apt-get install -y nodejs nodejs-legacy git wget build-essential python libcairo2-dev libgif-dev libpango1.0-dev libjpeg62-turbo-dev fonts-dejavu libboost-filesystem-dev libboost-program-options-dev libboost-regex-dev libboost-system-dev libboost-thread-dev libgdal-dev libicu-dev libpq-dev libcurl4-gnutls-dev libproj-dev libtiff-dev libwebp5 apt-transport-https && rm -rf /var/lib/apt/lists/*
-RUN echo > /etc/apt/sources.list && echo deb "https://apt.wikimedia.org/wikimedia jessie-wikimedia backports" >> /etc/apt/sources.list
-RUN apt-get update && apt-get install -y --force-yes -t jessie-wikimedia libmapbox-variant-dev libmapnik-dev mapnik-utils && rm -rf /var/lib/apt/lists/*
+FROM debian:stretch
+RUN apt-get update && apt-get install -y nodejs nodejs-legacy git wget python libcairo2-dev libjpeg62-turbo-dev libpango1.0-dev libgif-dev build-essential g++ libmapnik-dev && rm -rf /var/lib/apt/lists/*
 ENV NVM_DIR /usr/local/nvm
 RUN wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.6/install.sh | bash && . $NVM_DIR/nvm.sh && nvm install 6.11.1
 ENV HOME=/root/ LINK=g++
 ENV IN_DOCKER=1
 COPY . /home/code
 WORKDIR /home/code
-CMD . $NVM_DIR/nvm.sh && nvm use 6.11.1 && npm install --build-from-source=mapnik --fallback-to-build=false && npm test
+CMD . $NVM_DIR/nvm.sh && nvm use 6.11.1 && npm install && npm test
```

Comment:
Won't this now pull the binary blob for Mapnik?

Reply:
Yes, that was what I assumed was happening with these flags removed.  It builds and the tests pass in this case, which suggests to me that possibility (1) from https://phabricator.wikimedia.org/T188674 (using binary blobs in prod) is viable now, on Stretch, if we want to go that route.  But I'll play with it some more while keeping the flags in place.

Reply:
Backed out this part of the change.